sdk/core/azure-core/src/main/java/com/azure/core/models/spatial/GeometryPosition.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.models.spatial;
+
+import java.util.Objects;
+
+/**
+ * Represents a geometric position.
+ */
+public final class GeometryPosition {
+    private final double longitude;
+    private final double latitude;
+
+    private final Double altitude;
+
+    /**
+     * Constructs a geometric position.
+     *
+     * @param longitude Longitudinal position.
+     * @param latitude Latitudinal position.
+     */
+    public GeometryPosition(double longitude, double latitude) {
+        this(longitude, latitude, null);
+    }
+
+    /**
+     * Constructs a geometric position.
+     *
+     * @param longitude Longitudinal position.
+     * @param latitude Latitudinal position.
+     * @param altitude Altitude position.
+     */
+    public GeometryPosition(double longitude, double latitude, Double altitude) {
+        this.longitude = longitude;
+        this.latitude = latitude;
+        this.altitude = altitude;
+    }
+
+    /**
+     * The longitudinal position of the geometric position.
+     *
+     * @return The longitudinal position.
+     */
+    public double getLongitude() {
+        return longitude;
+    }
+
+    /**
+     * The latitudinal position of the geometric position.
+     *
+     * @return The latitudinal position.
+     */
+    public double getLatitude() {
+        return latitude;
+    }
+
+    /**
+     * The altitude of the geometric position.
+     *
+     * @return The altitude.
+     */
+    public Double getAltitude() {
+        return altitude;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
We encountered situations in user studies where users seem to be wanting a `toString()` method on the returned models. I am considering adding those in FormRecognizer, do we think we should consider that here too?

Reply:
Could be done, I was thinking it could be their GeoJSON representation.